internal/cli/clients/install_client.go

```diff
@@ -376,3 +390,22 @@ func (c *InstallClient) fetchIP(ctx context.Context, ip *string) error {
 
 	return nil
 }
+
+func (c *InstallClient) traefikServiceIngressInfo(ctx context.Context) (string, error) {
+	serviceList, err := c.kubeClient.Kubectl.CoreV1().Services("").List(ctx, metav1.ListOptions{
+		LabelSelector: "app.kubernetes.io/name=traefik",
+	})
+	if err != nil {
+		return "", err
+	}
+	if len(serviceList.Items) == 0 {
+		return "not found", nil
```

Comment:
I made no effort to try to detect a Traefik which is not deployed by Epinio. I check the labels of the one installed by `k3d` and thy don't match the label above. I'm not sure if it's worth trying to detect previously installed Traefik.